syntax = "proto3";

package test;

service MyService {
  rpc MyMethod (MyRequest) returns (MyResponse);
}

message MyRequest {
    string message_content = 1;
}

message MyResponse {
  repeated string serialized_data = 1;
}

